### PR TITLE
IBX-6709: Page Builder amendments / Drag and drop elements

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-6709)
| Versions      | master
| Edition       | Experience, Commerce

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by dkorostynski — APPROVED** [review]

File: docs/content_management/create_edit_pages.md [hunk 1]
```diff
@@ -39,71 +40,135 @@ Switch to the Field view using the Page Builder toolbar.
...
 
-![Page Builder Toolbar](img/page_builder_toolbar.png)
+A. Drop zone
 
-Using the toolbar buttons, you can perform the following actions:
+B. Elements / Structure view toolbar
+
+C. Settings toolbar (including Fields, Visibility and Schedule settings)
+
+D. Mode toolbar (including Desktop, Tablet and Mobile mode)
```
Comment (Gengar-i) [inline docs/content_management/create_edit_pages.md:62]: Maybe Device toolbar?

**Review by Gengar-i — CHANGES_REQUESTED** [review]

File: docs/content_management/create_edit_pages.md [hunk 2]
```diff
@@ -5,29 +5,30 @@ edition: experience
 
 # Create and edit Pages
 
-**Page** is a special Content Type that contains zones onto which you can drop 
-different dynamic blocks.
-You can edit pages to personalize the layout and contents of your website.
+**Page** is a special Content Type.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/create_edit_pages.md:8]: Not in the scope of this PR, will be covered in separated PR.

File: docs/content_management/create_edit_pages.md [hunk 3]
```diff
@@ -5,29 +5,30 @@ edition: experience
...
+You can create and modify Pages with a visual drag-and-drop editor - Page Builder.
+Page is divided into zones into which you can drop various dynamic blocks.
+By editing pages you can customize the layout and content of your website.
 
 Whenever you edit a Page, a [new version](content_versions.md) is created in the repository.
 
 !!! tip
-    The Page Content Type contains a **Landing Page** Field Type which manages 
+    The Page Content Type contains a **Landing Page** Field Type which manages
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/content_management/create_edit_pages.md:16]: Not in the scope of this PR, will be covered in separated PR.

**Review by dew326 — APPROVED** [review]

**Review by Gengar-i — APPROVED** [review]

File: docs/content_management/create_edit_pages.md [hunk 4]
```diff
@@ -5,29 +5,30 @@ edition: experience
 
 # Create and edit Pages
 
-**Page** is a special Content Type that contains zones onto which you can drop 
-different dynamic blocks.
-You can edit pages to personalize the layout and contents of your website.
+**Page** is a special Content Type.
+You can create and modify Pages with a visual drag-and-drop editor - Page Builder.
```
Comment (dabrt) [inline docs/content_management/create_edit_pages.md:9]:
```suggestion
You can create and modify Pages with a visual drag-and-drop editor, the Page Builder.
```

File: docs/content_management/create_edit_pages.md [hunk 5]
```diff
@@ -5,29 +5,30 @@ edition: experience
...
+Page is divided into zones into which you can drop various dynamic blocks.
+By editing pages you can customize the layout and content of your website.
```
Comment (dabrt) [inline docs/content_management/create_edit_pages.md:11]:
```suggestion
Pages are divided into zones into which you can drop various dynamic blocks.
By editing pages, you can customize the layout and content of your website.
```

**Review by dabrt — COMMENTED** [review]
Make sure you merge in changes from previous PRs and add your changes again.

**Pull request closed without merging**